canonical-web-and-design/maas-ui
# refactor(kvm): update KVM details to RRv6

## Done

- Update the single LXD and virsh details to React Router v6.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the KVM LXD tab and click on a single host. Each tab should work.
- Go back to the KVM list and click on the Virsh tab.
- Click on a Virsh host and each tab should work.

## Fixes

Fixes: #4191.
